### DISCUSSION TITLE
Error running Swift example project

### DISCUSSION THREAD

**Discussion by citruslab** [WalletConnect/walletconnect-monorepo#4345] [Developer Support]
Hello! I'm trying to run the Web3Modal Swift example project but I get these Auth Token creation errors in Xcode's console. What am I missing? 🙏  I'm building to my iPhone and entered walletconnect project ID in configuration file. The app runs on the device, then the console returns:

---------------------------------------------------------------------------------
---------- 🧊 Atlantis is running (version 1.24.0)
---------- Github: https://github.com/ProxymanApp/atlantis
---------------------------------------------------------------------------------
[Atlantis] Looking for Proxyman app in the network...
Auth token creation error: OSStatus: -34018, message: A required entitlement isn't present.
Auth token creation error: OSStatus: -34018, message: A required entitlement isn't present.
Socket connection status: disconnected
Socket connection status: disconnected
Auth token creation error: OSStatus: -34018, message: A required entitlement isn't present.
Auth token creation error: OSStatus: -34018, message: A required entitlement isn't present.
connectionFailed
Socket connection status: disconnected

**Comment by glitch-txs** [comment]
Hi, could you move this issue here: https://github.com/WalletConnect/web3modal-swift/issues

Thanks!
